### PR TITLE
Add Less

### LINKED ISSUE #4250: Less icon
**Name:** LESS
**Website:** http://lesscss.org/
**Alexa rank:** 163836
  

**Official resources for icon and color:** http://lesscss.org/
  
  

**Note:**
Even if the rank is not awesome, LESS remain massively used and actively maintained: https://github.com/less/less.js

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Less

**Issue:** Closes #4250
**Alexa rank:** [~171k](https://www.alexa.com/siteinfo/lesscss.org)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Cleaned up a few imperfections in the "l" and the "s"s. Colour from [website](http://lesscss.org/) stylesheet.

### CODE REVIEW

File: icons/less.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...84-.277-1.209-.506a.465.465 0 0 1-.224-.421c0-.183.084-.324.239-.422.154-.098.365-.14.646-.14.506 0 1.026.126 1.574.379l.365-.956c-.562-.28-1.208-.421-1.939-.421zm4.512 0c-.675 0-1.21.14-1.589.421-.38.281-.576.689-.576 1.209 0 .422.112.773.351 1.026.24.253.619.478 1.153.688.042.015.14.057.295.113.45.183.759.31.914.436a.592.592 0 0 1 .238.478c0 .224-.084.393-.253.506-.168.112-.407.168-.716.168a3.72 3.72 0 0 1-.984-.155 3.904 3.904 0 0 1-.886-.337l-.14 1.04c.506.296 1.18.436 2.038.436.702 0 1.265-.155 1.686-.464.394-.309.59-.759.59-1.335 0-.436-.126-.787-.379-1.054s-.632-.492-1.166-.689c-.392-.153-.842-.277-1.209-.506a.465.465 0 0 1-.225-.421c0-.183.085-.324.24-.422.154-.098.364-.14.646-.14.506 0 1.026.126 1.574.379l.337-.956c-.562-.28-1.209-.421-1.94-.421zm-9.46.014c-.842 0-1.503.267-1.995.815-.492.548-.73 1.279-.73 2.192 0 .956.252 1.687.772 2.22.52.535 1.237.802 2.165.802.8 0 1.49-.183 2.08-.52l-.197-.984a3.66 3.66 0 0 1-1.813.492c-.492 0-.886-.155-1.167-.45-.28-.295-.435-.716-.45-1.25h3.852v-.591c0-.829-.225-1.49-.661-1.982-.45-.491-1.054-.744-1.855-.744zm-.013.983c.38 0 .674.127.885.38.211.253.323.618.323 1.082H7.67c.042-.492.182-.857.407-1.096.253-.239.548-.366.9-.366Z"/></svg>
```
Comment (adamrusted) [inline icons/less.svg:1]: There's a slight difference across the underside of most of the icon (see below with PR in red) which may just be a precision thing, and I assume you intentionally removed the bit of the `L` that hangs over the bracket?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/less.svg (hunk 1 above)
Comment (PeterShaggyNoble) [reply icons/less.svg:1]: Are you using the AI or the SVG for comparison, @adamrusted? I'm not seeing that difference when comparing it to the AI.

And, yes, the "overhang" of the `l` was something I deliberately removed as, in the original files, it's obscured by a rectangle so I figured it shouldn't be included.

Comment (adamrusted) [reply icons/less.svg:1]: Yep - I was using the AI, but just compared again and they do match up. Must've just been how the alignment was with the overhang on the L causing issues. Will merge this one now.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PeterShaggyNoble 💯

**Pull request merged**